index.js

```diff
@@ -27,8 +27,11 @@ module.exports = (options = {}) => {
 	directory = pkgDir.sync(directory);
 
 	if (directory) {
-		if (!isWritable(path.join(directory, 'node_modules'))) {
-			return undefined;
+		const nodeModules = path.join(directory, 'node_modules');
+		if (!isWritable(nodeModules)) {
+			if (fs.existsSync(nodeModules) || !isWritable(path.join(directory))) {
```

Comment:
Sorry if this is not the wanted style.  I can combine the two if statements if desired, I'm not sure what you want for maximum line length or how you want an `if` conditional split over multiple lines.